charts/nginx-ingress/values.yaml

```diff
@@ -501,6 +501,14 @@ rbac:
   ## Configures RBAC.
   create: true
 
+  clusterrole:
+    ## Create ClusterRole
+    create: true
+
+  clusterrolebinding:
+    ## Create ClusterRoleBinding
+    create: true
+
```

Comment:
Hi @hafe 
Would there ever be a scenario where someone would set:
1. `clusterrole.create=true` and `clusterrolebinding.create=false`
2. `clusterrole.create=false` and `clusterrolebinding.create=true`

Do you think it makes sense to just have `clusterrole.create` that will create both `ClusterRole` and `ClusterRoleBinding` resources?

Reply:
I think clusterrbac.create makes more sense if you want to combine the two.

Reply:
agree with `clusterrbac.create`, in this case combining these args into one is better because they're pretty much hard-coded to each other